Kernel/Storage/StorageManagement.h

```diff
@@ -45,6 +72,8 @@ class StorageManagement {
 private:
     bool boot_argument_contains_partition_uuid();
 
+    void enumerate_all_attached_storage_controller(bool force_pio, bool poll);
```

Comment:
`enumerate_attached_storage_controllers` ? Ambivalent on keeping `all` but this should end in `s`